cnerepo/cap-ec2
# feat: upgrade ruby, fix contact_point [CNE-8144]

## What this does
* Fixes some lingering discrepancies with the way aws-sdk-v1 did things and they way they're done now.
* Upgrades the ruby version to 2.7.2 in line with other apps we're upgrading
* Bumps the version

## How to test
* Checkout michael/upgrade-ruby-CNE-8144 on player-backend
* `bundle install`
* Ensure `cap sandbox hl:status` and `cap sandbox hl:deploy rev=michael/upgrade-ruby-CNE-8144` work as expected

# ¡Before merging!:

- [ ] Remove the `:branch => 'michael/upgrade-ruby-CNE-8144'` from the player-backend branch
- [ ] Tag the commit
